src/main/resources/com/google/api/codegen/csharp/gapic_snippets.snip

```diff
@@ -0,0 +1,229 @@
+@extends "csharp/common.snip"
+
+@snippet generate(api)
```

Comment:
I think "snippets" would be a more natural name for the object here. To resolve the naming conflict with the snippet function named "snippets", I recommend renaming that to "renderSnippets".